examples/old/robust_uncertainty.jl

```diff
@@ -26,14 +26,15 @@ N = ceil((2+2log(2/𝛿))^2) + 1
 M = rand(d,d)
 Σhat = 1/(d-1)*(M-ones(d)*μhat')'*(M-ones(d)*μhat')
 
-m = Model(solver=SCSSolver(eps=1e-8))
+# m = Model(with_optimizer(SCS.Optimizer, eps=1e-8))
+m = Model(with_optimizer(SCS.Optimizer))
 
-@variable(m, Σ[1:d,1:d], SDP)
+@variable(m, Σ[1:d, 1:d], PSD)
 @variable(m, u[1:d])
 @variable(m, μ[1:d])
 
 @constraint(m, norm(μ-μhat) <= Γ1(𝛿/2,N))
-@constraint(m, vecnorm(Σ-Σhat) <= Γ2(𝛿/2,N))
+@constraint(m, opnorm(Σ-Σhat) <= Γ2(𝛿/2,N))
```

Comment:
I'm not sure if this is the correct way of doing it. 

```jullia
julia> @constraint(m, opnorm(Σ-Σhat) <= Γ2(𝛿/2,N))
ERROR: MethodError: no method matching real(::JuMP.GenericAffExpr{Float64,VariableRef})
Closest candidates are:
  real(::Missing) at missing.jl:79
  real(::Complex) at complex.jl:63
  real(::Real) at complex.jl:77
  ...
Stacktrace:
 [1] opnorm2(::Array{JuMP.GenericAffExpr{Float64,VariableRef},2}) at /Users/osx/buildbot/slave/package_osx64/build/usr/share/julia/stdlib/v1.0/LinearAlgebra/src/generic.jl:495
 [2] opnorm(::Array{JuMP.GenericAffExpr{Float64,VariableRef},2}, ::Int64) at /Users/osx/buildbot/slave/package_osx64/build/usr/share/julia/stdlib/v1.0/LinearAlgebra/src/generic.jl:556
 [3] opnorm(::Array{JuMP.GenericAffExpr{Float64,VariableRef},2}) at /Users/osx/buildbot/slave/package_osx64/build/usr/share/julia/stdlib/v1.0/LinearAlgebra/src/generic.jl:555
 [4] top-level scope at /Users/chkwon/.julia/packages/JuMP/LjMor/src/macros.jl:329
```

Reply:
The norm syntax is not supported anymore, you should do `[Γ2(𝛿/2,N); vec(Σ-Σhat)] in SecondOrderCone()`. Same for the previous line, it should be `[Γ1(𝛿/2,N); μ-μhat] in SecondOrderCone()`.